packages/block-editor/src/components/inserter/quick-inserter.js

```diff
@@ -70,13 +73,10 @@ export default function QuickInserter( {
 		}
 	}, [ setInserterIsOpened ] );
 
-	const { __unstableSetInsertionPoint } = useDispatch( blockEditorStore );
-
 	// When clicking Browse All select the appropriate block so as
 	// the insertion point can work as expected
 	const onBrowseAll = () => {
-		__unstableSetInsertionPoint( rootClientId, blockIndex );
-		setInserterIsOpened( true );
```

Comment:
I found it also strange that we'd have to set the insertion point, while no insertion point should show. I guess that's why the show/hide actions were added.